src/commands/Admin/config.ts

```diff
@@ -146,32 +147,39 @@ export class ConfigCommand extends BirthdayySubcommand {
 	) {
 		settings ??= {};
 		const t = await fetchT(interaction);
+		if (interaction.guild === null) throw new Error('Guild not found');
```

Comment:
this should never happen in theory!

Reply:
It will never happen you can delete it

Reply:
it is done that we dont have to use ? after interaction.guild later.
![image](https://github.com/BirthdayyBot/BirthdayyBot/assets/58025175/dd9c5c7e-2898-443e-bd65-78614c6dc34f)